README.md

```diff
@@ -1,17 +1,5 @@
-# js-ipfs-ceramic
-Wraps js-ipfs instance with dag-jose codec enabled.
+# ⛔️ DEPRECATED
 
-## Usage
+### Up to date versions are now included in [js-ceramic](https://github.com/ceramicnetwork/js-ceramic/tree/develop/packages/ipfs-daemon) and named `ipfs-daemon`
```

Comment:
The full package name in NPM is `@ceramicnetwork/ipfs-daemon`, which is probably worth including here